docs/operations/versioning.md

```diff
@@ -0,0 +1,305 @@
+
+Because Shoot clusters using an `unmaintained` minor version potentially have a higher security risk, Gardener will update these clusters after a certain date / grace-period.
+By defining the field `expirationDurationUnmaintainedVersion` in the Gardener Controller Manager configuration ([see below](#gardener-controller-manager-configuration-for-version-maintenance)), Gardener can set a different expiration date for 
+`unmaintained` versions than for `maintained` versions (the expiration date for `maintained` versions can be configured by the field `controllers.cloudProfile.kubernetesVersionManagement.expirationDuration`). 
+For example `maintained` versions are deprecated with an expiration of four months, while `unmaintained` already expire in one. 
+As described above, expired versions will be forcefully updated by Gardener.
+
+### Gardener Controller Manager configuration for Version Maintenance
+
+The Gardener Controller Manager has to be configured to enable the optional `Version Maintenance`.
+This is an example Gardener Controller Manager configuration enabling `Version Maintenance` for Kubernetes, but not for Machine image versions:
+
+``` yaml
+apiVersion: controllermanager.config.gardener.cloud/v1alpha1
+kind: ControllerManagerConfiguration
+controllers:
+  cloudProfile:
+    concurrentSyncs: 5
+    kubernetesVersionManagement:
+      expirationDuration: 2880h # defaults to 120 days / 4 months
+      versionMaintenance:
+        maintainedKubernetesVersions: 3
+        expirationDurationUnmaintainedVersion: 720h
+    machineImageVersionManagement:
+      expirationDuration: 2880h # defaults to 120 days / 4 months
+```
+
+## Duties of the Gardener Operator
+
+Even though Gardener strives for high automation, we think that it leads to safer operations 
+having a human operator in charge of the following operations: 
+
+- Adding `preview` versions
+- Promotion of a `preview` version to `supported`. We think this should be done on a case-by-case basis, hence Gardener deliberately does not offer an automation in this case.
+- Deprecate a `supported` version
+   - This is of particular importance when using `Version Management`. As it defines an initial classification for versions with no classification, 
+ it can happen that it tries to introduce another `supported` version even though there is already one present. That is forbidden and blocked by the Gardener API server. 
+ That situation occurs when specifying an unclassified version that is the overall latest version (or the latest version of a minor version when using `Version Maintenance`).
+ It is the operator duty to make sure that this does not happen. Specifically when introducing a new latest version without a classification, the current `supported` version should be manually deprecated ahead of time.
```

Comment:
```suggestion
 It is the operator's duty to make sure that this does not happen. Specifically when introducing a new latest version without a classification, the current `supported` version should be manually deprecated ahead of time.
```

Reply:
> should be manually deprecated ahead of time.

Does this mean that the deprecation date has to be entered manually as well?

Reply:
No.